kentikapi/errors.go

```diff
@@ -0,0 +1,99 @@
+package kentikapi
+
+import (
+	"errors"
+
+	kentikError "github.com/kentik/community_sdk_golang/kentikapi/internal/errors"
+)
+
+// IsAuthError checks if passed error is Authorization error, if yes return true.
+func IsAuthError(err error) bool {
+	var ktErr interface {
```

Comment:
```suggestion
	var ke kentikErrors.KentikError
```
There is no need to look for any error with Code() method. We can simplify it to look for specific KentikError.
In other places too.

Reply:
Done